doc/man1/ciphers.pod

```diff
@@ -346,7 +346,7 @@ Cipher suites using SHA1.
 
 =item B<SHA256>, B<SHA384>
 
-Ciphersuites using SHA256 or SHA384.
+Cipher suites using SHA256 or SHA384.
```

Comment:
Well, as in many things there is a whole vocabulary which is fairly specialist and not likely to appear in a dictionary. In this case the word comes from the RFC - although both cipher suite and ciphersuite seems to be used.